gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx

```diff
@@ -0,0 +1,40 @@
+---
+date: '2021-11-30'
+title: Synapse 1.48.0 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+[Synapse 1.48.0](https://github.com/matrix-org/synapse/releases/tag/v1.48.0) is out now!
+> __NOTE:__ Synapse 1.49, due out on December 14th, will be the **last release of Synapse to support Python 3.6 or PostgreSQL 9.6** per our [platform dependency deprecation policy](https://matrix-org.github.io/synapse/v1.48/deprecation_policy.html). Accordingly, we will remove support for Ubuntu 18.04 LTS (Bionic) at the same date, as it ships with Python 3.6.
```

Comment:
Hm. GitHub dropped a newline here. Let's add it back.

```suggestion

> __NOTE:__ Synapse 1.49, due out on December 14th, will be the **last release of Synapse to support Python 3.6 or PostgreSQL 9.6** per our [platform dependency deprecation policy](https://matrix-org.github.io/synapse/v1.48/deprecation_policy.html). Accordingly, we will remove support for Ubuntu 18.04 LTS (Bionic) at the same date, as it ships with Python 3.6.
```

Reply:
Way ahead of you :stuck_out_tongue: https://github.com/matrix-org/matrix.org/pull/1130#discussion_r760370938